src/components/va-accordion/va-accordion-item.css

```diff
@@ -0,0 +1,45 @@
+@import '../../mixins/focusable.css';
```

Comment:
`postcss` will inline this. Using imports like this can allow us to have shared styling across components, since this normally doesn't work due to the shadow DOM.